src/cli/commands/snip.test.ts

```diff
@@ -0,0 +1,48 @@
+import { Stats } from "fs";
+import { System } from "../../bluehawk";
+import { snip } from "./snip";
+
+describe("snip", () => {
+  it("snips", async (done) => {
+    System.fs = {
```

Comment:
This was already updated on the latest main branch, so be sure to fetch and rebase.